src/index.ts

```diff
@@ -3,8 +3,10 @@ Debug.enable("impftermin:*");
 import puppeteer from "puppeteer-core";
 import { loadConfiguration } from "./configuration";
 import { SOUND_BASE64 } from "./sound.base64";
+import { SOUND_RoadRunner_BASE64 } from "./soundRoadRunner.base64";
 import { sendTelegramMessage } from "./telegram";
 import { checkForUrlWithCode } from "./zentrum";
+import { bookAppointment } from "./appointments";
```

Comment:
Could you extract you feature into a separate file please? Like `bookings` or something? That would clear up the appointments file.

Reply:
Yes, if you prefer. I thought it would fit logically into appointments, because it is the last step of the process which is covered with the functions in appointments.ts

Reply:
yes it logically fits into the appointments file. However, I would prefer splitting these features cause both scripts are getting pretty long